proto/ethermint/evm/v1/evm.proto

```diff
@@ -32,104 +26,99 @@ message ChainConfig {
   // Homestead switch block (nil no fork, 0 = already homestead)
   string homestead_block = 1 [
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
-    (gogoproto.moretags) = "yaml:\"homestead_block\""
+    (gogoproto.moretags)   = "yaml:\"homestead_block\""
```

Comment:
@crypto-facs Are you using a different linter for proto? Looks like it formats everything different than our other proto files

Reply:
I ran `make proto-all` and that runs `proto-format`

Reply:
you should run `make proto-gen`

Reply:
or open another PR with the formating changes